modules/mysql/options.go

```diff
@@ -0,0 +1,34 @@
+package mysql
+
+type Option func(*Config)
+
+type Config struct {
+	username   string
+	password   string
+	database   string
+	configFile string
+}
+
+func WithUsername(username string) Option {
+	return func(config *Config) {
+		config.username = username
+	}
+}
+
+func withPassword(password string) Option {
```

Comment:
Export this option for users
```suggestion
func WithPassword(password string) Option {
```